GNUmakefile

```diff
@@ -43,7 +45,7 @@ cdl.a: lib/3rdparty/cdl.c lib/3rdparty/cdl.h
 libbemenu.so: private override LDLIBS += -ldl
 libbemenu.so: lib/bemenu.h lib/internal.h lib/filter.c lib/item.c lib/library.c lib/list.c lib/menu.c lib/util.c cdl.a
 
-bemenu-renderer-curses.so: private override LDLIBS += `pkg-config --libs ncurses` -lm
+bemenu-renderer-curses.so: private override LDLIBS += `pkg-config --libs ncurses || if [ -r /usr/lib/libncursesw.so ]; then echo /usr/lib/libncursesw.so; else echo /usr/lib/libncurses.so; fi` -lm
```

Comment:
This is *insane*.

Reply:
Very constructive.

Reply:
Don't do this. Instead use the `pkg-config` and provide a .pc file.

Reply:
Example of a .pc file you can write and add to search path with `PKG_CONFIG_PATH`, could include the working file in the repo too for freebsd, if there's no package or nobody's gonna package such.
```
prefix=/usr
exec_prefix=${prefix}
libdir=${prefix}/lib
includedir=${prefix}/include

Name: ncurses
Description: ncurses library
Version: whatever
Libs: -lncurses
```